### PR TITLE
Add new test for empty observable

### PR DESCRIPTION
**Pull request by YevhenLysen**

### CODE REVIEW

File: tests/functional/tests/test_empty_observebles.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from ctrlibrary.core.utils import get_observables
+from ctrlibrary.threatresponse.enrich import enrich_observe_observables
+from tests.functional.tests.constants import MODULE_NAME
+
+
+def test_positive_smoke_empty_observable(module_headers):
+    """Perform testing for enrich observe observables endpoint to
+    check observables is returned empty from Spycloud module
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observebles.py:8]: ... to check that observable, on which Spycloud doesn't have information, will return empty data

**Review by GeorgiyII — APPROVED** [review]

File: tests/functional/tests/test_empty_observebles.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    Importance: Critical
+    """
+    observable = [{'type': 'email', 'value': 'marita67@example.org'}]
+    response_from_all_modules = enrich_observe_observables(
+        payload=observable,
+        **{'headers': module_headers}
+    )
+    response_from_spycloud_module = get_observables(
+        response_from_all_modules['data'], MODULE_NAME)
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observebles.py:27]: I think we need add extra variable here which will be call 
`spycloud_data = response_from_all_modules['data']`
 send it to get_obs method, for adding in future `spycloud_errors` and make code more readable.

File: tests/functional/tests/test_empty_observebles.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+from tests.functional.tests.constants import MODULE_NAME
+
+
+def test_positive_smoke_empty_observable(module_headers):
+    """Perform testing for enrich observe observables endpoint to
+    check that observable, on which Spycloud doesn't have information,
+    will return empty data
+
+
+
```
Comment (mistergreen22) [inline tests/functional/tests/test_empty_observebles.py:12]: pls remove 2 extra lines

**Review by mistergreen22 — APPROVED** [review]
